lib/utils/logger.js

```diff
@@ -93,6 +94,8 @@ Object.freeze(methods);
  * @param {*} method
  */
 const log = method => (message, deviceId) => {
+	const currentTimestamp = timestamp('YYYYMMDDTHHmmss');
```

Comment:
@jan-tomasek-iflix could you please update timestamp format to `YYYY.MM.DD HH:mm:ss` and wrap it to square brackets? So that resulting string would be something like `[2018.08.14 12:40:53]`.
One of our devs will make format configurable through CLI params and RC file after this pull request is merged, so you'll still be able to use timestamp in your preferred format.